compose/ui/ui-text/src/skikoMain/kotlin/androidx/compose/ui/text/SkiaParagraph.skiko.kt

```diff
@@ -260,6 +261,16 @@ internal class SkiaParagraph(
         }
     }
 
+    private fun getAlignedStartingPosition(isRtl: Boolean): Float =
+        when (layouter.textStyle.textAlign) {
+            TextAlign.Center -> width / 2
+            TextAlign.Start -> if (isRtl) width else 0f
+            TextAlign.Right -> width
+            TextAlign.Left -> 0f
+            TextAlign.End -> if (isRtl) 0f else width
```

Comment:
Just nitpick - sort enum values as they defined:

```suggestion
            TextAlign.Left -> 0f
            TextAlign.Right -> width
            TextAlign.Center -> width / 2
            TextAlign.Start -> if (isRtl) width else 0f
            TextAlign.End -> if (isRtl) 0f else width
```